lib/dotcom_web/templates/fare/proposed_sales_locations.html.heex

```diff
@@ -57,14 +57,14 @@
       <p>
         We're asking the public to help guide our decisions as we roll out the Fare Transformation project over the next several years.
       </p>
-      <%= link "Find out how to get involved", to: "https://faretransformation.mbta.com/get-involved", target: "_blank", class: "c-call-to-action" %>
+      <%= link "Find out how to get involved", to: "https://www.mbta.com/fares/fare-transformation/get-involved", target: "_blank", class: "c-call-to-action" %>
```

Comment:
**suggestion**: let's use the helper function to generate this URL, so the hostname can change based on which environment the website's being served from. e.g use `cms_static_page_path/2`.

Reply:
Do we want to do the same thing with the [retail_sales_locations](https://github.com/mbta/dotcom/blob/main/lib/dotcom_web/templates/fare/retail_sales_locations.html.heex#L42)

Reply:
If it needs it, yeah that'd be great!

Reply:
It looks like [`cms_static_page_path/2`](https://github.com/mbta/dotcom/blob/main/lib/dotcom_web/views/helpers.ex#L134) currently is just a pass through and has no impact on the generated URL.  

Though changing this to `cms_static_page_page(..., "/fares/fare-trans...")` will achieve the desired outcome as the path is relative which will generate a relative link